src/issue.ts

```diff
@@ -279,12 +279,29 @@ function getTypes(i: ApiIssue) {
 }
 
 function isPullRequest(i: ApiIssue) {
-  return !!i.PullRequest;
+  return !!i.isPr;
```

Comment:
If there's an `isPr` field now, I think we can just drop this function and use that instead